uris.go

```diff
@@ -86,22 +93,38 @@ func probablyFileURI(candidate *url.URL) bool {
 	return true
 }
 
-// copied from sourcegraph/go-langserver/util.go
 func pathHasPrefix(s, prefix string) bool {
+	return rawHasPrefix(s, prefix, "/")
+}
+
+func filepathHasPrefix(s, prefix string) bool {
+	return rawHasPrefix(s, prefix, string(os.PathSeparator))
+}
+
+// adapted from sourcegraph/go-langserver/util.go
+func rawHasPrefix(s, prefix, pathSep string) bool {
 	var prefixSlash string
-	if prefix != "" && !strings.HasSuffix(prefix, string(os.PathSeparator)) {
-		prefixSlash = prefix + string(os.PathSeparator)
+	if prefix != "" && !strings.HasSuffix(prefix, pathSep) {
+		prefixSlash = prefix + pathSep
```

Comment:
I am aware that Windows accepts `/` as a path separator in PowerShell and cmd (?), but it is safe to assume it will always return `\`.
And file URIs always use `/`.